src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/listener/TierCalculationEventListener.kt

```diff
@@ -25,13 +28,19 @@ class TierCalculationEventListener(
   }
 
   private fun readMessage(wrapper: String?): CalculationEventData {
-    val (message) = objectMapper.readValue(wrapper, Message::class.java)
-    return objectMapper.readValue(message, CalculationEventData::class.java)
+    val message = objectMapper.readValue(wrapper, SQSMessage::class.java)
```

Comment:
You could probably still use destructuring here to get the message and the id out e.g. 
`val (message, messageId) = objectMapper.readValue(wrapper, Message::class.java)`

Reply:
done